tests/test_vms_exist.py

```diff
@@ -24,6 +27,16 @@ def test_expected(self):
         for test_vm in WANTED_VMS:
             self.assertIn(test_vm, vm_set)
 
+    @unittest.skipIf(CONFIG["environment"] != "prod", "Skipping on non-prod system")
```

Comment:
As a future improvement, instead of skipping entirely non non-prod systems, you could have those tests assert that features.get("internal") is != 1.